lib/settings_base.py

```diff
@@ -600,6 +600,7 @@ def JINJA_CONFIG():
             'js/zamboni/buttons.js',
             'js/zamboni/tabs.js',
             'js/common/keys.js',
+            'js/common-min.js',
```

Comment:
common-min.js is is a generated file. It shouldn't be included here. In fact, it's generated from this list of files, so I'm surprised that this helps.